eng/pipelines/libraries/run-test-job.yml

```diff
@@ -50,8 +50,8 @@ jobs:
       # To run the tests we just send to helix and wait, use ubuntu hosted pools for faster providing and to not back up our build pools
       pool: ${{ parameters.pool }}
 
-      # Disable component governance if requested or on musl machines where it does not work well
-      ${{ if or(eq(parameters.disableComponentGovernance, true), eq(parameters.osSubGroup, '_musl')) }}:
```

Comment:
This .yml doesn't have a disableComponentGovernance parameter and nothing sets it so I removed it

Reply:
Is that a result of the refactoring? I added a bunch of these in #84683 (though not to this file)

Reply:
I haven't checked but could be yeah.